presto-hive/pom.xml

```diff
@@ -216,6 +216,11 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>commons-codec</groupId>
```

Comment:
We are trying to avoid using apache libraries. Why is this library needed? Can you achieve the same by using the codecs from Guava?